frontend/vite.config.js

```diff
@@ -28,6 +28,11 @@ export default defineConfig({
         target: 'http://localhost:8080',
         changeOrigin: true,
         secure: false
+      },
```

Comment:
Does the page load with port 5173 ? We had issues with logging in on 5173

Reply:
Yes it does, thats what I used to test the frontend. I believe there was one route (/survey) that wasn't properly configured to proxy to the backend port, but it worked after I fixed that.